src/index.d.ts

```diff
@@ -197,10 +197,10 @@ declare namespace connect {
      * See: https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_GetAttachment.html
      * @param args The arguments of the operation.
      */
-    downloadAttachment(args: DownloadAttachmentArgs): Promise<ParticipantServiceResponse<DownloadAttachmentResult>>;
+    downloadAttachment(args: DownloadAttachmentArgs): Promise<Blob>;
```

Comment:
What's this Blob used for?

Reply:
Blob returned is the downloaded attachment content from S3. This downloadAttachment method handles calling Participant service's GetAssignment and downloading the attachment from S3.